colour/volume/macadam_limits.py

```diff
@@ -142,3 +143,197 @@ def is_within_macadam_limits(
     simplex = np.where(simplex >= 0, True, False)
 
     return simplex
+
+
+def macadam_limits(target_brightness, illuminant=()):
```

Comment:
See how we describe that kind of parameter here: https://github.com/colour-science/colour/blob/develop/colour/volume/spectrum.py#L259